### PR TITLE
Documentation for "How do I install my operator"

### PR DESCRIPTION
**Pull request by anik120**

### CODE REVIEW

**Comment by ecordell** [comment]
/lgtm

I think we'll want additional instructions for "directly applying" without a subscription or a catalog, but that can come later.

File: docs/how-do-i-install-my-operator-with-olm.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+spec:
+  channel: stable
+  name: my-operator
+  source: my-catalog
+  sourceNamespace: olm
+ ``` 
+
+Once you have the subscription yaml, `kubectl apply -f Subscription.yaml` to install your operator. 
+
+You can inspect your Subscription with `kubectl get subs <name-of-your-subscription> -n <namespace>`.
```
[the remark this answers is not in the document]
Comment (anik120) [reply docs/how-do-i-install-my-operator-with-olm.md]: I think keeping it as a template will get the message across faster than using the example name/namespace.

File: docs/how-do-i-install-my-operator-with-olm.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+spec:
+  channel: <channel-you-want-to-subscribe-to>
+  name: <name-of-your-operator>
+  source: <name-of-catalog-operator-is-part-of>
+  sourceNamespace: <namespace-that-has-catalog>
+ ``` 
+
+For example, if you want to install an operator named `my-operator`, from a catalog named `my-catalog` that is in the namespace `olm`, and you want to subscribe to the channel `stable`, your subscription yaml would look like this: 
+
+```
```
[the remark this answers is not in the document]
Comment (anik120) [reply docs/how-do-i-install-my-operator-with-olm.md]: That's a neat hack! Thanks @awgreene!

**Comment by anik120** [comment]
@awgreene addressed your comments.

File: docs/how-do-i-install-my-operator-with-olm.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# How do I install my operator with OLM? 
+
+[Once you've made your operator available in a catalog](https://), [or you've chosen an operator from an existing catalog](https:// ), you can install your operator by creating a Subscription to a specific channel.
```
Comment (awgreene) [inline docs/how-do-i-install-my-operator-with-olm.md]: Nit: You can install your operator by just applying a CSV. The Subscription is used to subscribe to updates as they become available. Is this a distinction we should make?

Comment (anik120) [reply docs/how-do-i-install-my-operator-with-olm.md]: That is a good question @awgreene and looks like @ecordell mentioned that too.

The was I see it, we're taking the reader through a particular workflow of installing an operator. As such, subscribing to updates as they become available should be the workflow that's encouraged in the [basic use case section](https://github.com/operator-framework/olm-book/blob/master/index.md#basic-use-cases).

Installing an operator by just applying the CSV sounds like a handy workflow for an operator developer. It'll definitely be helpful to call this out somewhere. I'm thinking the [Advanced use cases section](https://github.com/operator-framework/olm-book/blob/master/index.md#advanced-use-cases)

Thoughts?

Comment (awgreene) [reply docs/how-do-i-install-my-operator-with-olm.md]: > The was I see it, we're taking the reader through a particular workflow of installing an operator. As such, subscribing to updates as they become available should be the workflow that's encouraged in the basic use case section.

This is a good point. For now I'm happy with the state of the PR and we can revisit where to highlight a direct install in the future.

**Pull request merged**